### DISCUSSION TITLE
Add Telegram to "Social Media" settings

### DISCUSSION THREAD

**Discussion by Varhal** [forem/forem#15386] [Feature Requests]
Hi! Telegram is a very popular messenger on the russian Internet. There people communicate, read news feeds, join interest groups, and so on. And accordingly, each project has a community in this messenger. 

The addition of Telegram will help the community grow.

![1637067690744](https://user-images.githubusercontent.com/41826142/141991010-bfbb5c1a-4a24-4ccd-b1b5-b1fae7e37007.png)

**Comment by Pierce846712** [comment]
Hi, how you are adding it could you guide me because I have to do it for basketball organization so that they could implement [marketing ideas](https://basketballist.com/basketball-marketing-ideas/). As I tried by watching videos from youtube but all in vain. Haven't found any proper source so that I could implement properly.
